tests/fips/openssl/openssl_fips_alglist.pm

```diff
@@ -46,6 +46,7 @@ sub check_pk_algos {
     my ($openssl_binary) = @_;
 
     my @valid_algos = qw(RSA rsa DSA dsa EC DH HMAC CMAC);
+    push(@valid_algos, 'ED', 'ML') unless is_sle('<16');
```

Comment:
I'm not sure if https://build.opensuse.org/request/show/1270033 would also be submitted to SLMicro branch, or if there's a preferred way to handle different versions